<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>230</y>
      <width>711</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Zašifrovat</string>
    </property>
    <widget class="QLineEdit" name="inputZasifrovat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Zadejte text:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zasifrovat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Zašifrovat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>63</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="outputZasifrovat">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>391</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>711</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Dešifrovat</string>
    </property>
    <widget class="QLineEdit" name="inputDesifrovat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Zadejte text:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="desifrovat">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Dešifrovat</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>63</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>→</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="outputDesifrovat">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>70</y>
       <width>391</width>
       <height>81</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>711</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Klíč</string>
    </property>
    <widget class="QLineEdit" name="klic">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Zadejte klíč:</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="error">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>70</y>
       <width>291</width>
       <height>61</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>40</y>
       <width>63</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="matrix">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>300</y>
      <width>270</width>
      <height>270</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="textElideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderHighlightSections">
     <bool>false</bool>
    </attribute>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
